middleware/access_log.go

```diff
@@ -241,6 +246,7 @@ func AccessLoggerWithConfig(config LoggerConfig) echo.MiddlewareFunc {
 					return buf.WriteString(strconv.FormatInt(res.Size, 10))
 				case "request_body":
 					if len(reqBody) > 0 {
+						reqBody, _ = MaskSensitiveInfo(reqBody, config.MaskedParameters)
```

Comment:
if this function only being used in this package
```suggestion
						reqBody, _ = maskSensitiveInfo(reqBody, config.MaskedParameters)
```